gateway/platforms/vertx3/vertx3/src/test/java/io/apiman/gateway/platforms/vertx3/components/jdbc/suitetests/QueryTest.java

```diff
@@ -79,7 +79,7 @@ public static void setupTable(TestContext context) {
                         "    BYTEARRAY varbinary(100) NOT NULL,\n" +
                         "    PRIMARY KEY (ID));";
                 String insertSql = "insert into APIMAN (ID, STRING, SHORT, INTEGER, LONG, DOUBLE, BOOLEAN, DATETIME, BYTEARRAY)\n" +
-                        "     VALUES  (1, 'Voltigeur', 11, 9001009, 101010101, 3.14159, true, '1976-06-29 00:00:00', 31), ";
```

Comment:
Mystery how this didn't cause fatal errors / test failures in the past .

Reply:
Must be because we coincidentally pulled in newer transitives for with the OkHttp update that it's now triggering proper failures in the tests 🤔